blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,76 @@
+:root {
```

Comment:
Checking the designs, I realized that this block seems to be `full-width`. If you check the video component, the max-width of it is 1170px, meanwhile the features carousel is bigger.
<img width="752" alt="image" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/3090999/d8151f63-411d-436d-9046-eeba6ebacb74">

Reply:
The full width for this container is different if you check the [designs ](https://www.figma.com/file/yv0VtHTvMoGPqkv0RAYcVp/PDP?node-id=3781%3A47982&mode=dev)
<img width="391" alt="Screenshot 2023-08-28 at 15 58 21" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/4208604/d64a81f2-18c6-416f-b5a7-63c2942b3ac8">
Not same as video component

Reply:
We have some issues here, in the designs I can see all the cards, but in the implementation I can only see 5. So the width of the content might not be correct.